app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -35,11 +35,30 @@ def create_job_template
     my_signal(minimize_indirect, :post_ansible_run, err.message, 'error')
   end
 
+  def translate_credentials!(launch_options)
+    %i[credential vault_credential cloud_credential network_credential].each do |cred_type|
+      credential_id = launch_options.delete("#{cred_type}_id".to_sym)
+      next if credential_id.blank?
+
+      launch_options[cred_type] = Authentication.find(credential_id).native_ref
```

Comment:
Should we rescue `ActiveRecord::RecordNotFound`?

Reply:
I don't think it's necessary. At least it wasn't the first 3 times this method was written :laughing: